ROADMAP.md

```diff
@@ -1,16 +1,37 @@
-# KServe 2023 Roadmap
+# KServe 2024-2025 Roadmap
+## Objective: "Support GenAI inference"
+- LLM Serving Runtimes
+   * Support Speculative Decoding with vLLM runtime.
+   * Support LoRA adapters.
+   * Support LLM Serving runtimes for TensorRT-LLM, TGI and provide benchmarking comparisions.
+   * Support multi-host, multi-GPU inference runtime.
+
+- LLM Autoscaling
+   * Support Model Caching with automatic PV/PVC provisioning.
+   * Support Autoscaling settings for serving runtimes.
+   * Support Autoscaling based on custom metrics.
+
+- LLM RAG Pipeline Orchestration
+   * Support declarative RAG with KServe Inference Graph.
+
+- GenAI Task APIs
```

Comment:
```suggestion
- Open Inference Protocol extension to GenAI Task APIs
```
What about making more explicit that this point is about Open Inference Protocol?